rust-actix/src/api/api_structs.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Mutex;
+
+use chrono::serde::ts_seconds;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+pub struct LatestAction {
+    pub latest: Mutex<i32>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Latest {
+    pub latest: i32,
+}
+impl Default for Latest {
+    fn default() -> Self {
+        Latest { latest: -1 }
+    }
+}
+
+#[derive(Deserialize)]
+pub struct MessageAmount {
+    pub no: i32,
+}
+impl Default for MessageAmount {
+    fn default() -> Self {
+        MessageAmount { no: 100 }
+    }
+}
+
+#[derive(Deserialize)]
+pub struct RegisterInfo {
+    pub username: String,
+    pub email: String,
+    pub pwd: String,
+}
+
+#[derive(Serialize)]
+pub struct RegisterError {
+    pub status: i32,
+    pub error_msg: String,
+}
+
+#[derive(Serialize)]
+pub struct Message {
```

Comment:
no text or content?

Reply:
the content is here V - in the frontend they named it text and in the API content but is the same thing cause it is related with the message model (text field); We can change during refactor.